src/components/ChartBar/ChartBar.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './ChartBar.css';
+import { max, barColors } from '../../config.js';
+
+function ChartBar(props) {
+  const { data, increaseBudget, decreaseBudget, order } = props;
+  const state = {
+    dragging: false,
+    lastMouseY: null,
+    updatedValue: data.amount
+  };
+
+  const getHeight = () => state.updatedValue/max * 100 + '%';
+
+  const updateHeight = e => {
+    const delta = state.lastMouseY-e.clientY;
+    const domElement = document.getElementById(`chartBar-${data.id}`);
+    const dollarsPerPixel = Math.floor(max / domElement.clientHeight);
+    delta > 0 
+      ? increaseBudget(delta * dollarsPerPixel, data.id) 
+      : decreaseBudget(delta * -1 * dollarsPerPixel, data.id);
```

Comment:
does changing this to "delta + dollarsPerPixel" help?

Reply:
yes it does, thanks for the catch!!!